packages/playwright-test/.npmignore

```diff
@@ -1 +1,2 @@
-src
\ No newline at end of file
+src
```

Comment:
We should switch this file to allowlist!

Reply:
Yeah, in a separate patch